doc/src/content/xdocs/spec.xml

```diff
@@ -328,15 +328,19 @@
     <section>
       <title>Data Serialization</title>
 
-      <p>Avro data is always serialized with its schema.  Files that
-	store Avro data should always also include the schema for that
-	data in the same file.  Avro-based remote procedure call (RPC)
-	systems must also guarantee that remote recipients of data
-	have a copy of the schema used to write that data.</p>
-
-      <p>Because the schema used to write data is always available
-	when the data is read, Avro data itself is not tagged with
-	type information.  The schema is required to parse data.</p>
+      <p>Avro data does not include type information.
+  The binary encoding does not include field names, either. As a
+  result, the original schema of the writer is always required in
+  order to read Avro data accurately.</p>
```

Comment:
I think it would be better to say "correctly" rather than "accurately" because it isn't a matter of accuracy or exactness. It is either the same value or a wrong one.